app/AndroidManifest.xml

```diff
@@ -64,6 +64,7 @@
         android:label="@string/application_name"
         android:hardwareAccelerated="true"
         android:usesCleartextTraffic="true"
+        android:requestLegacyExternalStorage="true"
```

Comment:
Also, https://github.com/dimagi/commcare-android/blob/master/app/src/org/commcare/utils/MediaUtil.java#L500 BitmapFactory decodeFile fails if we try and decode the image chosen from gallery.

Reply:
We likely need to explicitly request permissions to the filepath beforehand the same way that we now do for things like camera widget capture. I think Shubham reviewed usage of File handles previously, but some of these places where raw paths are used likely didn't get caught.